frontend/src/app/components/containers/complaints/outcomes/animal-outcomes/edit-animal-outcome.tsx

```diff
@@ -374,6 +374,23 @@ export const EditAnimalOutcome: FC<EditAnimalOutcomeProps> = ({
     return isValid;
   };
 
+  const updateDrugAuthorizationFromInput = (drugAuthorization: DrugAuthorization | undefined, type: string) => {
```

Comment:
This function has me scratching my head, it looks very very similar to `updateDrugAuthorization` plus I can't seem to grok where the `isValid` is coming from.

Reply:
cleaned this up a bit to reduce duplication